--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -219,18 +219,9 @@
         set(IN_SOURCE_BUILD ON)
         set(BUILD_TESTING_PREV ${BUILD_TESTING})
         set(BUILD_TESTING OFF CACHE BOOL "Disable all tests in dependencies.")
-        # TODO: Use same BUILD_SHARED_LIBS for Aws Common Runtime dependencies.
-        # libcurl and aws-sdk-cpp-core may link to different libcrypto, which leads to some issues for shared build.
-        if (ENABLE_OPENSSL_ENCRYPTION)
-            set(BUILD_SHARED_LIBS_PREV ${BUILD_SHARED_LIBS})
-            set(BUILD_SHARED_LIBS OFF)
-        endif ()
         set(CRT_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS})
         add_subdirectory(crt/aws-crt-cpp)
         set(BUILD_TESTING ${BUILD_TESTING_PREV})
-        if (ENABLE_OPENSSL_ENCRYPTION)
-            set(BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS_PREV})
-        endif ()
     else ()
         include(AwsFindPackage)
         set(IN_SOURCE_BUILD OFF)
