diff -ur qt/qtwayland/src/hardwareintegration/compositor/linux-dmabuf-unstable-v1/linuxdmabuf.h qt-fixed/qtwayland/src/hardwareintegration/compositor/linux-dmabuf-unstable-v1/linuxdmabuf.h
--- qt/qtwayland/src/hardwareintegration/compositor/linux-dmabuf-unstable-v1/linuxdmabuf.h	2019-08-13 11:25:25.000000000 +0200
+++ qt-fixed/qtwayland/src/hardwareintegration/compositor/linux-dmabuf-unstable-v1/linuxdmabuf.h	2019-11-20 13:47:24.215999231 +0100
@@ -40,6 +40,8 @@
 #ifndef LINUXDMABUF_H
 #define LINUXDMABUF_H
 
+#include <array>
+
 #include "qwayland-server-linux-dmabuf-unstable-v1.h"
 
 #include <QtWaylandCompositor/private/qwayland-server-wayland.h>
